package.json

```diff
@@ -103,6 +103,7 @@
     "@angularclass/hmr": "1.2.0",
     "@angularclass/hmr-loader": "3.0.2",
     "angular-in-memory-web-api": "0.1.13",
+    "angular2-datatable": "^0.5.3",
```

Comment:
I'm very concerned about the use of this library. It does not appear to be maintained. It is using Angular 2.1. We are 2.2 and will be upgrading to the latest very soon. 
https://github.com/mariuszfoltak/angular2-datatable/blob/master/package.json

If you really want to use this repo, then you will need to update it and possibly maintain it via PRs or forking it.
OR
You may want to look for a more popular and well maintained library.